bugswarm/client/docker.py

```diff
@@ -113,7 +113,7 @@ def _image_exists_locally(image_tag):
 def _image_location(image_tag):
     assert image_tag
     assert isinstance(image_tag, str)
-    return DOCKER_HUB_REPO + ':' + image_tag
+    return DOCKER_HUB_CACHED_REPO + ':' + image_tag
```

Comment:
Need to handle the case where an image is not cached but is a part of our dataset.

Reply:
Got it. If it's not cached, then use ```DOCKER_HUB_REPO```?

Reply:
Yes.